css/styles.css

```
@@ -318,26 +376,77 @@ p {
 
 .phase-items {    
     list-style: none;
-    font-family: Poppins, Helvetica, sans-serif;
     font-size: 18px;
-    font-weight: 400;
     line-height: 28px;
     letter-spacing: 0;
-    text-align: left;
     color: #c2c3c5;
+    text-transform: capitalizeupp;
+}
+
+.phase-items>p::before {
+    content: url(/img/logos/icon_w24h24.png);
+    margin-right: 8px;
+}
+
+.r-map-btns-arrows {
+    border: none;
+    background-color: transparent;
+}
+
+.arrow-left-roadmap {
+    transform: rotate(90deg);
 }
 
-.r-map-list-btns {
+.arrow-right-roadmap {
+    transform: rotate(-90deg);
+}
+
+.r-map-btns-arrows:hover {
+    fill: #21E786;
+}
+/* How else can I recolor the color of the svg arrow when hovering (because this method doesn't work properly) or should I just apply different images? I would like to use only styling. */
+
+.r-map-list-btns-item {
     list-style: none;
     font-size: 20px;
     line-height: 28px;
     letter-spacing: 0;
-    text-align: left;
     text-transform: uppercase;
+    color: #888b8e;
+    border: none;
+    background-color: transparent;
+}
+
+.r-map-list-btns-item:hover {
+    color: #fff;
+    cursor: pointer;
+}
+
+.r-map-list-btns-item:hover.r-map-list-btns-item:before {
+    content: url("data:image/svg+xml;charset=UTF-8,<svg xmlns='http://www.w3.org/2000/svg' height='30' width='30'><circle cx='15' cy='15' r='15' fill='%2321E786' fill-opacity='0.2' /> <circle cx='15' cy='15' r='7.5' fill='%2321E786' /></svg>");
```

Whoa!) I want to say bravo for knowing all ways how to insert the image 👏🏻

If you want to inline SVGs, I recommend to do it with [URL-encoder](https://yoksel.github.io/url-encoder/).

It will not only escape (replace with url-safe) `#` by `%23`, but also [do some other transformations](https://github.com/yoksel/url-encoder/blob/master/src/js/script.js#L134-L148): 
* Escape quotes
* Escape opening and closing angle brackets (`<` and `>`)
* Escape these symbols `%()?[`
* Remove extra new lines

Otherwise browser may skip your image.

Another thing what you can do is moving this icons into sprite as well. Then you can insert them from HTML markup or use here in styles via `url()` function
```suggestion
    content: url("img/sprite.svg#active-point");
```
Or you can combine these ways, but I won't suggest this way, because it's difficult to support:
```css
    content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='30' width='30'%3E%3Cuse href='img/sprite.svg%23active-point'/%3E%3C/svg%3E")
```